gardening/demo/crm_lead.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="crm_lead_2" model="crm.lead">
+    <field name="team_id" ref="sales_team.team_sales_department"/>
+    <field name="stage_id" ref="crm.stage_lead1"/>
+    <field name="expected_revenue">250.0</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="date_open">2024-07-16 06:15:11</field>
+    <field name="won_status">pending</field>
+    <field name="description"><![CDATA[<p>Hi!</p><p>Can you rake&nbsp;my garden this week end?</p><p>See you.<br></p>]]></field>
+    <field name="name">Rake a garden</field>
+    <field name="priority">2</field>
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
doesn't seem that useful. Is it :thinking:

Reply:
:eyes:

Reply:
Yes, it is the number of stars on the CRM lead/opportunity. We should keep it